nulib/work-archiver
# Replace SVG elements from HTML email template with public S3 images

Veronica mentioned that the SVG elements were not loading properly on her Window email client so I decided to just replaced them with static images hosted in the public S3 bucket.

**Note:** terraform already applied on staging

### Steps to test

- Download a work archive on staging and check the email to make sure the images load properly
- That's it!

Here's a screenshot from web Outlook on my Windows machine (where the SVG problem was discovered):
![email](https://user-images.githubusercontent.com/1395707/220178442-73603064-5d3c-4687-be53-1b4523d9bef8.png)
